src/item/Tween.js

```diff
@@ -214,6 +256,40 @@ var Tween = Base.extend(Emitter, /** @lends Tween# */{
         return this;
     },
 
+    /**
+     * {@grouptitle Event Handling}
+     *
+     * Attaches an event handler to the tween.
+     *
+     * @name Tween#on
```

Comment:
@lehni suggested that we document this as Tween#onUpdate

Reply:
Ok, I made the change.

Reply:
About not documenting `Tween#update()` until we remove the need to call start, I agree with you.